superchain/configs/sepolia/base.toml

```diff
@@ -56,5 +56,5 @@ data_availability_type = "eth-da"
   DisputeGameFactoryProxy = "0xd6E6dBf4F7EA0ac412fD8b65ED297e64BB7a06E1"
   FaultDisputeGame = "0x8A9bA50a785c3868bEf1FD4924b640A5e0ed54CF"
   MIPS = "0xFF760A87E41144b336E29b6D4582427dEBdB6dee"
-  PermissionedDisputeGame = "0x3f5c770f17A6982d2B3Ac77F6fDC93BFE0330E17"
+  PermissionedDisputeGame = "0x593d20c4c69485b95d11507239be2c725ea2a6fd"
```

Comment:
Let's change this one to the checksummed version as well: `0x593D20C4c69485B95D11507239BE2C725ea2A6fD`

Reply:
Yup just updated, PTAL again